src/com/cloud/chatbot/todo/Event.java

```diff
@@ -0,0 +1,35 @@
+package com.cloud.chatbot.todo;
+
+
+
+/**
+ * Represents an event TODO, which has both a starting and ending timestamp.
+ */
+public class Event extends Deadline {
+    private String timestampStart;
```

Comment:
```suggestion
    private String startingTimestamp;
```
Would it make more sense to name the variable as startingTimestamp so it sounds more like a noun?

Reply:
Although I note that those aren't constants, I was actually going for this rule:

![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/44526554/8e9c8c14-816d-4137-b7ef-0fbf8d9fa0e7)

Ie `timestampStart` matching `timestampEnd`. I've experimented with similar naming schemes elsewhere. I guess I still haven't fully decided on which I prefer more.

I will keep this in mind, though I think once I update the type away from `String`, the new type will make it clear by its nature that those variables are meant to represent a timestamp, and I can just name the variables something like `start` and `end`.